api-get-object-file.go

```diff
@@ -69,7 +69,8 @@ func (c *Client) FGetObject(ctx context.Context, bucketName, objectName, filePat
 	}
 
 	// Write to a temporary file "fileName.part.minio" before saving.
-	filePartPath := filePath + objectStat.ETag + ".part.minio"
+	filePartPath := filePath + sum256Hex([]byte(objectStat.ETag)) + ".part.minio"
```

Comment:
`sum256Hex` was already defined in utils.go so I utilized it to reduce moving parts

Reply:
There's no other place where this path gets reconstructed, right?